tutorials/fundamentals/concat-string-array.md

```diff
@@ -0,0 +1,17 @@
+JavaScript's `join()` method is handy for turning elements in
+an array into a string. You can specify what you want the
+elements to be joined by, like  `-`, for example but it defaults to a space.
```

Comment:
Let's move this sentence to a 2nd separate paragraph and show an example of using `-` as a custom separator. Also "by, like  `-`, for example but it" is an awkward sentence.